### ISSUE TITLE
Folder with files are added but solution file is empty (C++)

### ISSUE THREAD

**Issue by fefas** [JeffreyGbeho/leetcode-tracker#4]
Hello there, I first would like to thank you for developing this leetcode tracker.

I started using it today but it seems it is not working quite correct. Here is my linked [repo](https://github.com/fefas/leetcode-tracker). I am solving leetcode with `C++` and the added solution file has `null` at the end as if it has not recognized the languange:

`83-RemoveDuplicatesfromSortedListnull`.

**Comment by fefas** [comment]
I could debug a bit ad fixed the problem. I copied and pasted this code into my leetcode page console:

```js
getLanguageExtension() {
    const language = JSON.parse(window.localStorage.getItem("global_lang"));
    return languageExtensions[language] || null;
}
```

Steps:

1. Executing `window.localStorage.getItem("global_lang")` returned `null`
2. Switched to `PHP` and `window.localStorage.getItem("global_lang")` returned `php`
3. Switched back to to `C++` and `window.localStorage.getItem("global_lang")` returned `cpp`
4. Clicked submit again and my linked repo was properly updated

**Comment by JeffreyGbeho** [comment]
Thanks for reporting this issue with a great description! This helps me a lot to improve the extension.

The bug was caused because when you sign up for the first time on LeetCode, your `global_lang` property in localStorage is not set up. It only gets set up when you change the language, as you discovered.
So I've added a complementary way to retrieve it.

Now, I've pushed a fix and updated the Chrome extension. There will be a short wait before the update is available.
